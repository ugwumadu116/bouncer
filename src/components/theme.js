import * as colors from "./color/color";
import * as fontsize from "./text/fontsize";
import * as fontweight from "./text/fontweight";
import * as lineheight from "./text/lineheight";
import * as letterspacing from "./text/letterspacing";
import * as btnSizes from "./button/button";
import * as btnState from "./button/btnState";
import * as inputsizes from './form/input/inputsizes'
import * as inputbuttonSize from './form/inputButton/inputbuttonsizes'
import * as selectSize from './form/select/selectSizes'


const theme = {
  colors,
  fontsize,
  fontweight,
  lineheight,
  letterspacing,
  btnSizes,
  btnState,
  inputsizes,
  inputbuttonSize,
  selectSize
};
export default theme;
