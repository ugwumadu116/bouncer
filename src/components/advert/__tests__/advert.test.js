import React from 'react'
import renderer from 'react-test-renderer'
import {ThemeProvider} from 'styled-components'
import theme from '../../theme'
import Advert from '../'


it('renders correctly', ()=>{
    const tree = renderer.create(
    <ThemeProvider theme={theme}>
      <Advert/>  
    </ThemeProvider>
    
    ).toJSON();
    expect(tree).toMatchSnapshot();
})




