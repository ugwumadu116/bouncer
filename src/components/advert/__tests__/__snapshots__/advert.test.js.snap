// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa dXihno"
>
  <div
    className="mex"
  >
    <div
      className="advert"
    >
      <img
        alt="product advert"
        src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575986836/bouncer-frontend/iphone_ad"
      />
    </div>
    <div
      className="advert"
    >
      <img
        alt="product advert"
        src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575986849/bouncer-frontend/oculus_rift_ad_nrpsc2.png"
      />
    </div>
    <div
      className="advert"
    >
      <img
        alt="product advert"
        src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575986827/bouncer-frontend/go_pro_ad_fumlul.png"
      />
    </div>
  </div>
</div>
`;
